src/cli/subcommands/archive_cmd.rs

```diff
@@ -69,6 +72,9 @@ pub enum ArchiveCommands {
         /// How many state-roots to include. Lower limit is 900 for `calibnet` and `mainnet`.
         #[arg(short, long, default_value_t = 2000)]
         depth: ChainEpochDelta,
+        /// Do not include any values reachable from this epoch
+        #[arg(short, long)]
+        diff: Option<ChainEpochDelta>,
```

Comment:
It's a bit confusing, doc says epoch while code says epoch delta

Reply:
Updated to `epoch-diff`.